src/main/java/frc/robot/subsystems/CoralEndEffector.java

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkLimitSwitch;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.SparkMaxConfig;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class CoralEndEffector extends SubsystemBase {
+    /** Creates a new CoralEndEffector. */
+    private static final int MOTOR_CURRENT_LIMIT = 30;
+    private static final double MOTOR_VOLTAGE_COMP = 10; // This sets a limit for voltage to 10 so it is repeatable
+                                                         // untill the battery dips below 10 volts
+
+    // Speeds
+    // TODO: set these to real speeds
+    private static final double INTAKE_SPEED = -1;
+    private static final double OUTTAKE_SPEED = 1;
```

Comment:
Coral intake and outtake should be about 0.2. We will tune, but "1" is way too high.